plugins/source/aws/resources/services/eventbridge/event_bus_rules.go

```diff
@@ -35,6 +35,13 @@ func EventBusRules() *schema.Table {
 				Type:     schema.TypeJSON,
 				Resolver: resolveEventbridgeEventBusRuleTags,
 			},
+			{
+				Name: "arn",
```

Comment:
use [transformers.WithPrimaryKeys](https://pkg.go.dev/github.com/cloudquery/plugin-sdk/transformers#WithPrimaryKeys) option and not the extra column.
Additionally, should drop the `event_bus_arn` column as duplicate

Reply:
`event_bus_arn` is the parent arn...

Reply:
Ah, sorry, then dismissing that part.
The transformation & ad-hoc columns still stand, though